docs/setup-docs.md

```diff
@@ -278,14 +278,15 @@ The "Leader" is simply the guardian who will input the configuration settings fo
 
 <img src="setup-docs-assets/guardian_select.png" alt="Guardian Select" width="500">
 
-## Leader
+## Setup Leader
 
-The leader inputs an additional couple parameters beyond his name and password:
+The setup leader inputs an additional couple parameters beyond his name and password:
 
 - Federation Name: whatever you want to name your federation
 - Number of Guardians: the n of the t/n, so 4 for a 3/4 federation
 - Network: use signet (mutinynet is a fork of signet with 30 second blocktimes)
 - Block Confirmations: Fedimint's consensus CANNOT handle bitcoin block reorganizations (it's impossible to invalidate or distinguish e-cash once issued) so we recommend at least 6 confirmations (meaning the fedimint's consensus blockheight will be current blockheight - 6) for any mainnet or production use, but if you're just using this for development you can set it to 1 or 2
+- Meta Fields: these are arbitrary key/value pairs that can be used to store any information you want about your federation. Client apps can use this information to attach chat servers, a federation icon, a welcome message, or anything else you want to display to the user when they first connect to the federation.
```

Comment:
Link to meta field docs? https://github.com/fedimint/fedimint/tree/master/docs/meta_fields